plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardFacetTest.java

```diff
@@ -46,17 +46,25 @@
   @Rule
   public TestProjectCreator appEngineProjectCreator =
       new TestProjectCreator().withFacetVersions(JavaFacet.VERSION_1_7, WebFacetUtils.WEB_25,
-          AppEngineStandardFacet.FACET_VERSION);
+          AppEngineStandardFacet.JRE7);
 
   @Test
   public void testStandardFacetExists() {
     Assert.assertEquals("com.google.cloud.tools.eclipse.appengine.facets.standard",
         AppEngineStandardFacet.ID);
-    Assert.assertEquals("1", AppEngineStandardFacet.VERSION);
+    Assert.assertEquals("JRE7", AppEngineStandardFacet.JRE7.getVersionString());
+    if (AppEngineStandardFacet.FACET.getVersions().size() > 1) {
```

Comment:
As there is a separate test in `AppEngineStandardFacetVersionChangeTests`, I'll remove this.  It is a bit confusing.